NeuroID/src/main/java/com/neuroid/tracker/service/NIDJobServiceManager.kt

```diff
@@ -13,23 +17,31 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
 import okhttp3.OkHttpClient
+import org.json.JSONArray
+import org.json.JSONObject
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import java.util.concurrent.TimeUnit
 
-object NIDJobServiceManager {
+class NIDJobServiceManager(
```

Comment:
Suggest reverting the NIDJobServiceManager class changes here and make this a separate ticket and PR.